packages/eslint-config-expo/utils/react.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  parserOptions: { ecmaFeatures: { jsx: true } },
+  plugins: ['react', 'react-hooks'],
```

Comment:
Use fully qualified names `"plugin:react/recommended", "plugin:react-hooks/recommended"`

Reply:
You mean like this?
```
  extends: [
    "plugin:react/recommended",
    "plugin:react-hooks/recommended"
  ],
```

It doesn't look like that's equivalent of `plugins: ['react', 'react-hooks'],`, e.g. I get this when running it on a project:
```
error    'React' must be in scope when using JSX        react/react-in-jsx-scope
```

I can turn it off with `'react/react-in-jsx-scope': 'off',`, but I don't know if there are any other rules it enables that we're not expecting.